src/Facebook/Facebook.php

```diff
@@ -586,4 +587,38 @@ public function videoToUpload($pathToFile)
     {
         return new FacebookVideo($pathToFile);
     }
+
+    /**
+     * Upload a video in chunks.
+     *
+     * @param int         $target           The id of the target node before the /videos edge.
+     * @param string      $pathToFile       The full path to the file.
+     * @param array       $metadata         The metadata associated with the video file.
+     * @param string|null $accessToken      The access token.
+     * @param int         $maxTransferTries The max times to retry a failed upload chunk.
+     * @param string|null $graphVersion     The Graph API version to use.
+     *
+     * @return array
+     *
+     * @throws FacebookSDKException
+     */
+    public function uploadVideo($target, $pathToFile, $metadata = [], $accessToken = null, $maxTransferTries = 5, $graphVersion = null)
+    {
+        $accessToken  = $accessToken ?: $this->defaultAccessToken;
+        $graphVersion = $graphVersion ?: $this->defaultGraphVersion;
+
+        $uploader = new FacebookResumableUploader($this->app, $this->client, $accessToken, $graphVersion);
+        $endpoint = '/'.$target.'/videos';
+        $file     = $this->videoToUpload($pathToFile);
+        $chunk    = $uploader->start($endpoint, $file);
```

Comment:
I don't like thoses alignment, the reason is that it may make the diff harder to read. Imagine you add a new `$foobarbaz` variable, you will need to change all other line without any added value, and the diff will show multiple lines change but in fact you only add one. :)